src/MultiMessageSender.sol

```diff
@@ -85,17 +90,13 @@ contract MultiMessageSender {
     /// will be refunded back to msg.sender, which requires caller being able to receive native token.
     /// Caller can use estimateTotalMessageFee() to get total message fees before calling this function.
 
-    /// @param _dstChainId is the destination chainId.
-    /// @param _target is the contract address on the destination chain.
-    /// @param _callData is the data to be sent to _target by low-level call(eg. address(_target).call(_callData)).
-    /// @param _expiration is the unix time when the message expires, zero means never expire.
-    function remoteCall(uint256 _dstChainId, address _target, bytes calldata _callData, uint64 _expiration)
-        external
-        payable
-        onlyCaller
-    {
+    /// @param _dstChainId is the destination chainId
+    /// @param _target is the target execution point on dst chain
+    /// @param _callData is the data to be sent to _target by low-level call(eg. address(_target).call(_callData))
+    function remoteCall(uint256 _dstChainId, address _target, bytes calldata _callData) external payable onlyCaller {
         /// @dev writes to memory for gas saving
-        address[] memory adapters = senderAdapters[_dstChainId];
+        address[] memory adapters = senderAdapters;
```

Comment:
This change will mean that we have a single bridge set, that will be used to broadcast to all destinations. The challenge with this is that 1) it might not be the case that all bridges support all the chains that Uniswap is/will be deployed on. 2) that all messages have to be broadcast through the entire bridge set. This might sound reasonable at this stage but might not be ideal as the number of bridges grows, and we might want flexibility on what subset to use that still achieves a quorum on the destination. 

I suggest we deal with this in a separate PR. I'll write up a solution on how we can address this.